### PR TITLE
Add unit test for DeploymentConfigNodeContributor.

### PR DESCRIPTION
**Pull request by sudhirverma**
Fix: #1110

### CODE REVIEW

File: test/k8s/deployment.test.ts [hunk 1]
```diff
@@ -73,6 +74,39 @@ suite('K8s/deployment', () => {
...
+            }
+        };
+
+        setup(async () => {
+            kubectl = await extension.kubectl.v1;
+        });
+
+        test('should able to get the children node of deployment Config', async () => {
+            if (kubectl.available) {
+                sandbox.stub(kubectl.api, 'invokeCommand').resolves({code: 0,
```
Comment (dgolovin) [inline test/k8s/deployment.test.ts]: Would be good to test with actual output here, so getChildren would return at least two nodes.

**Comment by dgolovin** [comment]
@sudhirverma please rebase on top of the master.

File: test/unit/k8s/deployment.test.ts [hunk 2]
```diff
@@ -98,6 +100,40 @@ suite('K8s/deployment', () => {
...
+                description: "",
+                displayName: "DeploymentConfigs",
+                label: "DeploymentConfigs",
+                manifestKind: "DeploymentConfig",
+                pluralDisplayName: "DeploymentConfigs"
+            }
+        };
+
+        setup(() => {
+            const api: k8s.API<k8s.KubectlV1> = {
```
Comment (dgolovin) [inline test/unit/k8s/deployment.test.ts]: @sudhirverma I was asking to do something like this. 
You stub external api, create object instance and test actual methods, not prototype.

Comment (sudhirverma) [reply test/unit/k8s/deployment.test.ts]: Thanks, Denis :). I will check why test are failing.

**Pull request merged**